content/en/cloudcraft/getting-started/use-filters-to-create-better-diagrams.md

```diff
@@ -0,0 +1,80 @@
+---
+title: How to use filters to create better diagrams
+kind: guide
+---
+
+While we are constantly improving the platform across the board, the number of components rendered at once for large environment diagrams can introduce performance and readability issues, making for a poor experience.
+
+To ensure you do not face these problems, we recommend using the **Filtered layout** feature by applying filters or excluding services when placing live components.
+
+As a bonus, building smaller diagrams makes managing them much more manageable, gives viewers more control over how they ingest information and does not overwhelm them.
```

Comment:
```suggestion
The number of components rendered at once for large environment diagrams can introduce performance and readability issues, making for a poor experience.

To avoid these issues, Cloudcraft recommends that you use the **Filtered layout** feature to apply filters, or exclude services when placing live components.

Building smaller diagrams makes managing them much easier. It also gives viewers more control over how they ingest information.
```